src/Confluent.Kafka/AdminClient.cs

```diff
@@ -428,6 +429,33 @@ private ListConsumerGroupsReport extractListConsumerGroupsResults(IntPtr resultP
             }).ToList();
         }
 
+        private ListOffsetsResult extractListOffsetsResults(IntPtr resultPtr)
```

Comment:
Rename this to `extractListOffsetsReports` and create a new class `ListOffsetReport`, return a list of that from here. ListOffsetReport will have same definition as ListOffsetsResult but will also have the Error field to indicate if any error received during execution of the overall operation.